src/main/java/uk/gov/companieshouse/mapper/email/DissolutionEmailMapper.java

```diff
@@ -34,4 +40,41 @@ public SuccessfulPaymentEmailData mapToSuccessfulPaymentEmailData(Dissolution di
 
         return successfulPaymentEmailData;
     }
+
+    public ApplicationAcceptedEmailData mapToApplicationAcceptedEmailData(Dissolution dissolution) {
+        final String applicantEmailAddress = dissolution.getCreatedBy().getEmail();
+        final String dissolutionReferenceNumber = dissolution.getData().getApplication().getReference();
+        final String companyNumber = dissolution.getCompany().getNumber();
+        final String companyName = dissolution.getCompany().getName();
+
```

Comment:
why store these in variables? Feels like we could remove 10 lines from the file if we don't store these values in memory.

Reply:
Had it all stored in variables for readability (imo) and because of all the chained methods. Though because this mapper is growing it sounds good to reduce that all.